lib/generators/rolemodel/saas/devise/templates/app/views/admin/users/index.html.slim

```diff
@@ -0,0 +1,27 @@
+header.header--index-action
```

Comment:
Does this match the default HTML we have for some of the other Devise generators? Probably something we'll want to iterate on further as we add more updates/better naming to RoleModel CSS